frontend_vue/src/components/base/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface BaseStateVariantType {
+  danger: unknown;
+  warning: unknown;
+  info: unknown;
```

Comment:
what's the unknown type? *? any reason not to make it string?

Reply:
oh! Nope absolutely no reason, it's a leftover!
I push a change and make them strings 🙏
thanks for catching it